src/FixerDefinition/CodeSampleInterface.php

```diff
@@ -26,4 +26,9 @@ public function getCode();
      * @return null|array
      */
     public function getConfiguration();
+
+    /**
+     * @return null|int
+     */
+    public function getRequiredVersion();
```

Comment:
Hmm, do you have an idea how to solve the problem?

We could extend the interface, and have a concretion that implements the extended interface, what do you think?